packages/debugger-ui/src/ui/index.js

```diff
@@ -0,0 +1,200 @@
+/* eslint-env browser */
+import './index.css';
+import blueIcon from './assets/blue-icon.png';
+import grayIcon from './assets/gray-icon.png';
+import orangeIcon from './assets/orange-icon.png';
+
+('use strict');
+(function() {
```

Comment:
Do we need this IIFE? Isn't parcel wrapping the bundled module?

Reply:
It is unnecessary, left it because this was present in the original script. Will change that in a sec.